componentLibraries/extensionLibrary/Connectivity/FMIWrapper.hpp

```diff
@@ -414,6 +422,25 @@ class FMIWrapper : public ComponentSignal
 
         delete mpTempDir;
     }
+
+
+    //! @brief Removes all illegal characters from the string, so that it can be used as a variable name.
+    //! @param [in out] rName String that will be modified
```

Comment:
I guess you copied this, in which case the original is also wrong. 
The rName is actually only input and should be called name
Also the todo note is not relevant in this case

Reply:
Should we not use the "r" prefix also for const references? It is still a reference. The `[in out]` is wrong though, and the `@returns` is missing. I will fix that.